frontend/javascripts/oxalis/view/right-border-tabs/segments_tab/segments_view.tsx

```diff
@@ -768,6 +781,365 @@ class SegmentsView extends React.Component<Props, State> {
     </>
   );
 
+  getToastForMissingLocations = (groupId: number) => {
+    const segmentsWithoutPosition = this.getSegmentsWithMissingLocation(groupId);
+    if (segmentsWithoutPosition.length > 0) {
+      console.log(`Segments with unknown positions: ${segmentsWithoutPosition}`);
+      return Toast.info(
+        <React.Fragment>
+          {segmentsWithoutPosition.length} meshes couldn't be loaded because the segment position is
+          unknown. The segment IDs were printed to the console.
+        </React.Fragment>,
+      );
+    }
+  };
+
+  getSetGroupColorMenuItem = (id: number): ItemType => {
+    const isEditingDisabled = !this.props.allowUpdate;
+    return {
+      key: "changeGroupColor",
+      disabled: isEditingDisabled,
+      label: (
+        <ChangeColorMenuItemContent
+          title="Change Group Color"
+          isDisabled={isEditingDisabled}
+          onSetColor={(color) => {
+            if (getVisibleSegmentationLayer == null) {
+              return;
+            }
+            this.setGroupColor(id, color);
+          }}
+          rgb={[0.5, 0.5, 0.5]}
+        />
+      ),
+    };
+  };
+
+  getLoadMeshesMenuItem = (id: number): ItemType => {
+    if (this.props.currentMeshFile != null) {
+      return {
+        key: "loadMeshesOfGroup",
+        label: (
+          <>
+            <CloudDownloadOutlined />
+            Load all meshes
+          </>
+        ),
+        children: [
+          {
+            key: "loadAdHoc",
+            label: (
+              <div
+                onClick={() => {
+                  if (this.props.visibleSegmentationLayer == null) {
+                    return;
+                  }
+                  this.handleLoadMeshesAdHoc(id);
+                  this.getToastForMissingLocations(id);
+                  this.handleSegmentDropdownMenuVisibility(id, false);
+                }}
+              >
+                ad hoc
+              </div>
+            ),
+          },
```

Comment:
you can extract this into a variable and reuse that in the else-branch. the differing keys can be unified (it's only important that the keys are unique in the surrounding array).